digitalocean/kubernetes/resource_kubernetes_cluster.go

```diff
@@ -121,9 +121,10 @@ func ResourceDigitalOceanKubernetesCluster() *schema.Resource {
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"day": {
-							Type:     schema.TypeString,
-							Optional: true,
-							Computed: true,
+							Type:         schema.TypeString,
+							Optional:     true,
+							Computed:     true,
+							ValidateFunc: validation.IsDayOfTheWeek(true),
```

Comment:
I did not validate this also works with capitalized weekdays. The provider docs list lowercase examples but in the case of database clusters I tested and both were accepted by the DO API.

Reply:
I think ignoring case here is correct, but in addition to days of the week, DOKS maintenance policies can also set `any` if the user has no preference. 

`validation.IsDayOfTheWeek` is just a thin wrapper around `validation.StringInSlice`. So we can use that here it support `any` as well.

https://github.com/hashicorp/terraform-plugin-sdk/blob/00fc746ade8f31a96a9030b867bedcff3365afa5/helper/validation/time.go#L13-L24

Reply:
Ah right, I missed that even though I saw it in the docs.

I updated it and amended my commit. Hope that's alright.